vocab_files/observable_property_concepts/plant-height.ttl

```diff
@@ -7,13 +7,16 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 <https://linked.data.gov.au/def/test/dawe-cv/0e9da717-6c8e-4194-9385-c995d54702e4>
     a skos:Concept ;
-    dcterms:source "Ecological Field Monitoring protocols - Condition Module, draft v0.1, 30/11/2021" ;
+    dcterms:source "Ecological Field Monitoring protocols - Condition Module, Recruitment module draft v0.1, 30/11/2021" ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:altLabel "tree height" ;
```

Comment:
Technically tree height is not an alternative label. It's a narrower concept. All trees are plants but not all plants are trees.

Reply:
Let's merge this and deal with it later if it ever becomes a problem.